src/pages/tasks/TaskShareDestinationSelectorModal.js

```diff
@@ -49,7 +50,13 @@ function TaskShareDestinationSelectorModal(props) {
     const filteredReports = useMemo(() => {
         const reports = {};
         _.keys(props.reports).forEach((reportKey) => {
-            if (!ReportUtils.isAllowedToComment(props.reports[reportKey]) || ReportUtils.isArchivedRoom(props.reports[reportKey])) {
+            console.log('reportKey', props);
+            if (
+                !ReportUtils.isAllowedToComment(props.reports[reportKey]) ||
+                ReportUtils.isArchivedRoom(props.reports[reportKey]) ||
+                (lodashGet(props.reports[reportKey], 'participantAccountIDs', []).length === 1 &&
+                    _.some(props.reports[reportKey].participantAccountIDs, (accountID) => _.contains(CONST.EXPENSIFY_ACCOUNT_IDS, accountID)))
```

Comment:
Make common function in ReportUtils similar to `isConciergeChatReport` and use it both here and ReportActionCompose.
We avoid duplicated code.

Reply:
Roger that